samples/cellular/nrf_cloud_multi_service/overlay_coap.conf

```diff
@@ -7,6 +7,7 @@
 # General config
 CONFIG_FPU=y
 CONFIG_NEWLIB_LIBC_NANO=n
+CONFIG_HEAP_MEM_POOL_SIZE=16384
```

Comment:
Is a similar change needed for the wifi-coap overlay, or is it already enough?

Reply:
Not sure. Might not need this change either, now that I removed the big modem info structure.